crates/ide/src/syntax_highlighting/tests.rs

```diff
@@ -915,6 +915,10 @@ fn main() {
 }
 
 #[test]
+#[cfg_attr(
+    all(unix, not(target_pointer_width = "64")),
```

Comment:
Contrary to what title says this runs this test on UNIXes that are **not** 64-bit 😄

Reply:
:scream_cat: no, it actually runs it on 32-bit non-Unix.

Reply:
Oh, sorry I thought it's normal `cfg`. Didn't see that `ignore` part 🤦🏻‍♂️

Reply:
Well, it's still wrong 😀, thanks for spotting it.